packages/core/esm/URI.js

```diff
@@ -0,0 +1,12 @@
+let URI = {
```

Comment:
might be const or just export default { ... }

Reply:
good catch. used to have an inject, but since I removed it and there is no eslint here, it slipped.